src/stores/models/Option.test.js

```diff
@@ -0,0 +1,238 @@
+import { observable } from 'mobx';
+import { Token, Phases, OracleStatus } from 'constants';
+import Option from './Option';
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+const { QTUM, BOT } = Token;
+const { BETTING, VOTING, FINALIZING } = Phases;
+
+describe('Option Model', () => {
+  const addr = {
+    address: 'qSu4uU8MGp2Ya6j9kQZAtizUfC82aCvGT1',
+    qtum: 2000,
+    bot: 100,
+  };
+  const app = observable({
+    sortBy: 'ASC',
+    wallet: { addresses: [addr] },
+    global: { syncBlockNum: 0 },
+    refreshing: false,
+    ui: { location: 0 },
+    eventPage: { selectedOptionIdx: 0 },
+  }); // mock the appstore
+  let store;
+  let oracle;
+  beforeEach(() => {
+    store = null;
+    oracle = {
+      txid: '0906a03d4ebb95258731970ef87cfc76b818e37aee35b0b6e17b9f620e0b5287',
+      amounts: [99, 100],
+      consensusThreshold: 100,
+      address: '02e91962156da21fae38e65038279c020347e4ff',
+      topicAddress: '4044f951857f2885d66d29a475235dacdaddea84',
```

Comment:
dont have to change this in this one, but lets use randomized string/numbers for any type of hex address or txid in the future.

Reply:
Harrison's testing PR has the crypto random string functionality so lets add that in a later PR after his is merged.

Reply:
oh yeah I was planning on that but forgot on the way....